server/game/cards/02_SHD/events/Outflank.ts

```diff
@@ -0,0 +1,23 @@
+import { EventCard } from '../../../core/card/EventCard';
+
+export default class Outflank extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '0802973415',
+            internalName: 'outflank',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Attack with a unit',
+            initiateAttack: {},
```

Comment:
nit: they both do the same thing but could we instead do `immediateEffect: AbilityHelper.immediateEffects.attack()`, ditto line 17. I think the meaning of that version is a little clearer in that case for someone new. passing the empty object here might be hard to understand for someone new to js or to the repo